index.html

```diff
@@ -97,6 +98,7 @@
     table.simple td {
         border-right: 1px solid #000;
     }
+    .issue[id^='issue-'] > *:not([role='heading']) {}
```

Comment:
On Tue, Sep 22, 2015 at 6:03 PM, Jeffrey Yasskin <notifications@github.com>
wrote:

> In index.html
> <https://github.com/w3c/web-nfc/pull/50#discussion_r40097227>:
>
> > @@ -97,6 +98,7 @@
> >      table.simple td {
> >          border-right: 1px solid #000;
> >      }
> > +    .issue[id^='issue-'] > *:not([role='heading']) {}
>
> Ah, I see. Then the display:none should come back, uncommented?
>

I have removed the whole line, since it's not used any more. Besides the
display:none would remove any text, but I wanted to be able to add my own
text for describing the issue. So it was good as it was when it had its use
- but again, now it's not needed any more.